tests/dmd-64/test.bat

```diff
@@ -6,6 +6,10 @@ set SLEEPY_SILENT_CRASH=1
 
 for %%b in (64) do (
 	echo Testing %%b
+
+	rem  Discard any saved config options.
+	call ..\..\scripts\clear_config

```

Comment:
Would it make sense to put this in the `tests/run_tests.bat` loop, so it's done before any test is run?

Reply:
I didn't want to fool with other tests that I couldn't (time constraints) run, like the alternate compilers or debuggers. If you'd like to see it there, I can put it there.